src/app.js

```diff
@@ -1,21 +1,25 @@
+/* eslint-disable no-console */
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
+const http = require('http');
+const PORT = process.env.PORT || 3007;
 
-module.exports = sum;
+const server = http.createServer((req, res) => {
+  res.setHeader('Content-Type', 'application/json');
+
+  const normalizedUrl = new URL(req.url, 'http:/' + req.headers.host);
+
+  console.log(normalizedUrl);
+
+  const parts = normalizedUrl.pathname.slice(1)
+    .split('/').filter(el => el.length > 0);
+  const query = Object.fromEntries(normalizedUrl.searchParams.entries());
+
+  res.end(JSON.stringify({
+    parts,
+    query,
+  }));
+});
+
+server.listen(PORT, () => {
+});
```

Comment:
```suggestion
server.listen(PORT);
```
If you don't do anything in callback you don't need it